### ISSUE TITLE
Installed *.exe wrappers are not symlinks

### ISSUE THREAD

**Issue by huangqinjin** [mstorsjo/msvc-wine#71]
*.exe in [wrappers](https://github.com/mstorsjo/msvc-wine/tree/master/wrappers) are symlinks, but they are installed by plain `cp` command, and thoses symlinks are resolved to normal files. Is it intentional? It not, `cp -a` should be used.
https://github.com/mstorsjo/msvc-wine/blob/2146cbfaf037e21de56c7157ec40bb6372860f51/install.sh#L136

**Comment by mstorsjo** [comment]
> *.exe in [wrappers](https://github.com/mstorsjo/msvc-wine/tree/master/wrappers) are symlinks, but they are installed by plain `cp` command, and thoses symlinks are resolved to normal files. Is it intentional? It not, `cp -a` should be used.

No, I don't recall a reason for that being intentional, thanks for bringing it up.

FWIW, the `cp` in older versions of macOS didn't use to support the `-a` flag, but I believe that it might have been added quite a long time ago (probably over 10 years ago), so it's probably quite fine to rely on it at this point.

I pushed https://github.com/mstorsjo/msvc-wine/commit/1272d55944e248b0aefab489e28b83064db125a5 on a branch, I can merge it to master if the CI looks good.
